### ISSUE TITLE
Error importing on Python 3.5.2

### ISSUE THREAD

**Issue by Undermmon** [tmbo/questionary#15]
I am using Questionary 1.2.0 with Python 3.5.2, and during the import i am getting this error:

```
Traceback (most recent call last):
  File "/home/undermon/Documentos/Desenvolvimento/Python/cli_tests/cli.py", line 1, in <module>
    import questionary
  File "/home/undermon/Documentos/Desenvolvimento/Python/cli_tests/venv_test/lib/python3.5/site-packages/questionary/__init__.py", line 7, in <module>
    from questionary.prompt import prompt
  File "/home/undermon/Documentos/Desenvolvimento/Python/cli_tests/venv_test/lib/python3.5/site-packages/questionary/prompt.py", line 8, in <module>
    from questionary.prompts import AVAILABLE_PROMPTS, prompt_by_name
  File "/home/undermon/Documentos/Desenvolvimento/Python/cli_tests/venv_test/lib/python3.5/site-packages/questionary/prompts/__init__.py", line 2, in <module>
    from questionary.prompts import text
  File "/home/undermon/Documentos/Desenvolvimento/Python/cli_tests/venv_test/lib/python3.5/site-packages/questionary/prompts/text.py", line 21, in <module>
    style: Optional[Style] = None,
  File "/usr/lib/python3.5/typing.py", line 552, in __getitem__
    dict(self.__dict__), parameters, _root=True)
  File "/usr/lib/python3.5/typing.py", line 512, in __new__
    for t2 in all_params - {t1} if not isinstance(t2, TypeVar)):
  File "/usr/lib/python3.5/typing.py", line 512, in <genexpr>
    for t2 in all_params - {t1} if not isinstance(t2, TypeVar)):
  File "/usr/lib/python3.5/typing.py", line 1077, in __subclasscheck__
    if super().__subclasscheck__(cls):
  File "/usr/lib/python3.5/abc.py", line 225, in __subclasscheck__
    for scls in cls.__subclasses__():
TypeError: descriptor '__subclasses__' of 'type' object needs an argument

Process returned 1 (0x1)	execution time : 0.265 s
```

**Comment by vaneseltine** [comment]
If it's an option for you, a workaround is upgrading Python to anything above 3.5.2: typing/issues/266

**Comment by vaneseltine** [comment]
This happens on import of Style from prompt_toolkit... I'm betting that this is due to their addition of type annotations in c7851c09f (6 May 2019).

**Comment by Undermmon** [comment]
Unfortunately i am not able to upgrade Python at the moment.

**Comment by vaneseltine** [comment]
Hmm, the problem was actually the line above `style: Optional[Style] = None`. 

#8 raised the same problem in April, but it looks like not all the uses of `Type` were caught in 70165f0

This is an exhaustive removal of `Type` from annotations, which I confirmed fixed the issue (3.5.2 install on Windows).
